src/Parser/LASParser.js

```diff
@@ -109,8 +110,16 @@ export default {
         const parsedData = await lasLoader.parseChunk(Transfer(data), {
             pointCount: options.in.pointCount,
             header: options.in.header,
-            eb: options.eb,
+            eb: options.in.eb,
             colorDepth: options.in.colorDepth,
+            in: {
+                crs: options.in.crs,
+                projDefs: proj4.defs(options.in.crs),
+            },
+            out: {
+                crs: options.out.crs,
+                projDefs: proj4.defs(options.out.crs),
+            },
```

Comment:
Factorize with the in line 154

Reply:
I don't really see the benefit of doing so, moreover when we have the idea (in the futur) to allows proj4.defs as crs. In this cas it will simplifed the options as passing only the crs.